sscanf_conv.go

```diff
@@ -0,0 +1,13 @@
+// Implementation using Sscanf
+package convert
+
+import "fmt"
+
+func convSScanf(s string) (int, error) {
+	var i int
+	if _, err := fmt.Sscanf(s, "%d ", &i); err != nil {
+		return 0, fmt.Errorf("Not correct input.")
```

Comment:
Please don't override an error, that returns some functions. Will be better just return this error.

Reply:
Ok, I'll fix it a little bit later

Reply:
Sorry, forgot about it, is it better now?

Reply:
Not yet, will be better:
`return 0, err`